rajawali/src/main/java/c/org/rajawali3d/engine/RenderModel.java

```diff
@@ -0,0 +1,44 @@
+package c.org.rajawali3d.engine;
+
+import android.support.annotation.Nullable;
+import c.org.rajawali3d.annotations.GLThread;
+
+/**
+ * Interface to be implemented by models that are to be displayed in RednerViews managed by the Engine
+ *
+ * Author: Randy Picolet
```

Comment:
Use the javadoc `@author` label please. It results in nice formatting by the documentation processors.